core/src/main/kotlin/org/evomaster/core/problem/rest/resource/ResourceCluster.kt

```diff
@@ -0,0 +1,83 @@
+package org.evomaster.core.problem.rest.resource
+
+import org.evomaster.client.java.controller.api.dto.database.operations.DataRowDto
+import org.evomaster.core.EMConfig
+import org.evomaster.core.database.SqlInsertBuilder
+import org.evomaster.core.database.schema.Table
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.search.Action
+
+class ResourceCluster {
```

Comment:
add comment to clarify what this class is for.

Reply:
the comment is added.